packages/angular/src/utils/versions.ts

```diff
@@ -1,15 +1,15 @@
 export const nxVersion = require('../../package.json').version;
 
-export const angularVersion = '~14.2.0';
-export const angularDevkitVersion = '~14.2.0';
-export const ngPackagrVersion = '~14.2.0';
-export const ngrxVersion = '~14.0.0';
+export const angularVersion = '~15.0.0';
+export const angularDevkitVersion = '~15.0.0';
+export const ngPackagrVersion = '~15.0.0';
+export const ngrxVersion = '~15.0.0-beta.0';
```

Comment:
If this won't become stable, should we stay on `~14.0.0`?

Reply:
I'm not sure. According to brandon, most features have landed it's just some polish, but i guess it might make more sense to wait for that stable release

Reply:
Dropped to 14